autowarefoundation/autoware
# Feature/ouster submodule

## Status
DEVELOPMENT

## Description
Updated Ouster LiDAR submodule head to incorporate recent changes.

## Related PRs
None

## Todos

## Steps to Test or Reproduce
(As this PR is proposed from a forked version of Autoware at `https://github.com/alexandrx/Autoware.git`)
1. Get a fresh copy of Autoware `develop` branch, ex. `git clone https://github.com/alexandrx/Autoware.git -b develop --recurse-submodules [AUTOWARE_PATH]`.
2. (OPTIONAL) Make sure the original builds correctly: run `./catkin_make_release` (follow the steps in [How to build](https://github.com/CPFL/Autoware/wiki/Source-Build) ).
3. Switch to `feature/ouster_submodule` branch, ex. `git checkout feature/ouster_submodule` and pull for updates `git pull`. Update all submodules `git submodule update --init`
4. Confirm the branch for Ouster driver is correctly set to `autoware_branch`:
```
cd ros/src/sensing/drivers/lidar/packages/ouster
git checkout autoware_branch
```
5. Go back to `[AUTOWARE_PATH]/ros` folder and make sure Autoware builds correctly: run `catkin_make -DCMAKE_BUILD_TYPE=Release`.
